fortran/simulated-annealing/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           fortran 1.0
+PortGroup           linear_algebra 1.0
+
+github.setup        jacobwilliams simulated-annealing 57e249338abb97cb23a51018ea9152f5dc5a6c43
+version             2024.01.23
+revision            0
+categories-append   science
+license             BSD
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Simulated Annealing with Modern Fortran
+long_description    {*}${description}
+checksums           rmd160  6356541490461c06f054fe818bcd637aa0c68d74 \
+                    sha256  879be1819147839b065ce64206aa0e3c146f428d480c3667c306ecf4de5e85ce \
+                    size    43140
+github.tarball_from archive
+
+set_default_variants    no
+
+if {![variant_isset accelerate] && ![variant_isset atlas] && ![variant_isset blis] \
```

Comment:
Could you wrap this at 80 columns? Yes, that probably means three lines instead of two.

Reply:
We are making it uglier than it could have been )